testcafe/tests/photos/create_full_album_scenario.js

```diff
@@ -1,96 +1,211 @@
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_ADD_EXISTING_PHOTOS}`)
+  await timelinePage.selectPhotos(1)
+  await timelinePage.checkCozyBarOnTimeline()
+  await t.click(selectors.btnAddToAlbumCozySelectionBar)
+
+  await t.expect(selectors.spinner.exists).notOk('Spinner still spinning')
+  await checkAllImagesExists()
+
+  await t.fixtureCtx.vr.takeScreenshotAndUpload({
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_ADD_EXISTING_PHOTOS}-1`,
+    withMask: maskPhotosCluster
+  })
+  await t.click(selectors.album(`New2_${FEATURE_PREFIX}`))
+
+  //Wait for toast alert to disapear before taking screenshot
+  await checkToastAppearsAndDisappears(
+    'This album already contains this photo.'
   )
+  //go to album to get the screenshot
   await timelinePage.goToAlbums()
-  await photoAlbumsPage.goToAlbum(`New2_${ALBUM_DATE_TIME}`)
-  await photoAlbumPage.removePhoto(1)
+  await photoAlbumsPage.goToAlbum(`New2_${FEATURE_PREFIX}`)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload({
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_ADD_EXISTING_PHOTOS}-2`,
+    delay: THUMBNAIL_DELAY,
+    pageToWait: photoAlbumPage
+  })
   console.groupEnd()
 })
 
-test('Go to New2_ALBUM_DATE_TIME, and delete it', async () => {
-  console.group(`↳ ℹ️  Go to New2_${ALBUM_DATE_TIME}, and delete it`)
+test(TEST_DELETE_PHOTO_FROM_TIMELINE, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_DELETE_PHOTO_FROM_TIMELINE}`)
+  await timelinePage.selectPhotosByName([IMG0])
+  await timelinePage.deletePhotosFromTimeline({
+    numOfFiles: 1,
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_DELETE_PHOTO_FROM_TIMELINE}-1`,
+    withMask: maskPhotosCluster
+  })
+  await t.fixtureCtx.vr.takeScreenshotAndUpload({
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_DELETE_PHOTO_FROM_TIMELINE}-2`,
+    delay: THUMBNAIL_DELAY,
+    pageToWait: timelinePage,
+    withMask: maskPhotosCluster
+  })
+  console.groupEnd()
+})
+
+test(TEST_REMOVE_PHOTO, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_REMOVE_PHOTO}`)
   await timelinePage.goToAlbums()
-  await photoAlbumsPage.goToAlbum(`New2_${ALBUM_DATE_TIME}`)
+  await photoAlbumsPage.goToAlbum(`New2_${FEATURE_PREFIX}`)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload({
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_REMOVE_PHOTO}-1`,
+    delay: THUMBNAIL_DELAY,
+    pageToWait: photoAlbumPage
+  })
+  await photoAlbumPage.removePhotoFromAlbum(1)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload({
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_REMOVE_PHOTO}-2`,
+    delay: THUMBNAIL_DELAY,
+    pageToWait: photoAlbumPage
+  })
+  console.groupEnd()
+})
+
+test(TEST_DELETE_ALBUM, async () => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_DELETE_ALBUM}`)
+  await timelinePage.goToAlbums()
+  await photoAlbumsPage.goToAlbum(`New2_${FEATURE_PREFIX}`)
   await photoAlbumPage.deleteAlbum()
   await photoAlbumPage.waitForLoading()
   await photoAlbumsPage.checkEmptyAlbum() //There is no more album
```

Comment:
Instead of `.checkEmptyAlbum` it should be `.hasNoAlbum` or `.isEmpty`. When I read `.checkEmptyAlbum` it feels like we test that an album is empty which isn't the case here